Docker instructions not working due to missing config values

# Story
_As a ServiceX developer I want to run the Docker image in a container so I can test it in isolation._

The current documentation for running the app as a Docker image does not seem to work. Steps to reproduce:
- Checkout the develop branch at the latest commit
- Build the Docker image
- Run it using the command in the README

The main culprit seems to be that `docker-dev.conf` is missing a number of required config values.

```
$ docker build . -t servicex_app
...
Successfully built c0028c455e65
Successfully tagged servicex_app:latest
$ docker run --name servicex-app --rm -p8000:5000  --mount type=bind,source="$(pwd)"/sqlite,target=/sqlite  -e APP_CONFIG_FILE=/home/servicex/docker-dev.conf  servicex_app:latest
[2021-04-21 11:24:11 +0000] [18] [INFO] Worker exiting (pid: 18)
[2021-04-21 11:24:11 +0000] [22] [ERROR] Exception in worker process
Traceback (most recent call last):
...
  File "/home/servicex/servicex/__init__.py", line 95, in create_app
    if app.config['OBJECT_STORE_ENABLED']:
KeyError: 'OBJECT_STORE_ENABLED'
[2021-04-21 11:24:11 +0000] [22] [INFO] Worker exiting (pid: 22)
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 209, in run
    self.sleep()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 357, in sleep
    ready = select.select([self.PIPE[0]], [], [], 1.0)
  File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 242, in handle_chld
    self.reap_workers()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 525, in reap_workers
    raise HaltServer(reason, self.WORKER_BOOT_ERROR)
gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
```

# Acceptance Criteria
1. The Docker image can be run locally by following the documentation.

--- a/docker-dev.conf
+++ b/docker-dev.conf
@@ -3,6 +3,7 @@
 # use a longer secret string known only to your application
 # details are beyond the scope of this example
 SECRET_KEY = 'abc123!'
+ENABLE_AUTH=False
 
 # Based on https://codeburst.io/jwt-authorization-in-flask-c63c1acf4eeb
 SQLALCHEMY_DATABASE_URI = 'sqlite:////sqlite/app.db'
@@ -10,4 +11,35 @@ SQLALCHEMY_TRACK_MODIFICATIONS = False
 SECRET_KEY = 'some-secret-string'
 JWT_SECRET_KEY = 'jwt-secret-string'
 RABBIT_MQ_URL= 'amqp://user:leftfoot1@host.docker.internal:30672/%2F'
-ADVERTISED_HOSTNAME: 'host.docker.internal:5000'
+ADVERTISED_HOSTNAME = 'host.docker.internal:5000'
+
+# Keep retrying every ten seconds for two minutes before giving up
+RABBIT_RETRIES = 12
+RABBIT_RETRY_INTERVAL = 10
+
+
+# This will be mounted into the transformer pod's /data directory
+TRANSFORMER_LOCAL_PATH="/Users/bengal1/dev/IRIS-HEP/data"
+TRANSFORMER_NAMESPACE="default"
+TRANSFORMER_X509_SECRET="servicex-x509-proxy"
+TRANSFORMER_PULL_POLICY="IfNotPresent"
+
+TRANSFORMER_VALIDATE_DOCKER_IMAGE = True
+
+ADVERTISED_HOSTNAME= 'host.docker.internal:5000'
+
+TRANSFORMER_MANAGER_ENABLED = False
+TRANSFORMER_AUTOSCALE_ENABLED = False
+
+TRANSFORMER_DEFAULT_IMAGE = "sslhep/servicex_func_adl_xaod_transformer:1.0.0-RC.3"
+
+TRANSFORMER_MESSAGING = 'none'
+OBJECT_STORE_ENABLED = False
+CODE_GEN_SERVICE_URL = 'http://localhost:5001'
+CODE_GEN_IMAGE = "sslhep/servicex_code_gen_func_adl_xaod:develop"
+
+DID_FINDER_DEFAULT_SCHEME = 'rucio'
+VALID_DID_SCHEMES = ['rucio']
+
+ELASTIC_SEARCH_LOGGING_ENABLED = False
+

